apps/fluent-tester/src/FluentTester/FluentTesterApp.tsx

```diff
@@ -2,14 +2,17 @@
 
 import { ThemeProvider } from '@fluentui-react-native/theme';
 import * as React from 'react';
+import { Platform } from 'react-native';
 import { FluentTester, FluentTesterProps } from './FluentTester';
 import { tests } from './testPages';
 import { testerTheme } from './theme/index';
 
+const isMobile = Platform.OS == 'android' || (Platform.OS == 'ios' && !Platform.isPad);
```

Comment:
Remove iOS for now. I realized I don't have a way to handle "going back" because BackHandler is android specific. I'll figure that out in a followup and re-add the platforms to this check.

Reply:
Yep removing iOS for now.